frontend/.gitignore

```diff
@@ -1,2 +1,4 @@
 public/*
 !public/index.html
+!public/favicon
```

Comment:
It might be useful to make this a bit more generic so we can store other images here. Perhaps we can rename the `favicon` directory to `images`?

Reply:
Sure, i can rename the folder images and update the `.gitignore`.